ui/page/about_page.go

```diff
@@ -129,6 +131,25 @@ func (pg *AboutPage) layoutRows(gtx layout.Context) layout.Dimensions {
 			})
 		},
 		func(gtx C) D {
+			if pg.licenseRow.IsHovered() {
+				return pg.shadowBox.Layout(gtx, func(gtx C) D {
+					return pg.licenseRow.Layout(gtx, func(gtx C) D {
+						return layout.Flex{}.Layout(gtx,
+							layout.Rigid(func(gtx C) D {
+								return in.Layout(gtx, pg.license.Layout)
+							}),
+							layout.Flexed(1, func(gtx C) D {
+								return layout.E.Layout(gtx, func(gtx C) D {
+									return in.Layout(gtx, func(gtx C) D {
+										pg.chevronRightIcon.Color = pg.Theme.Color.Gray1
+										return pg.chevronRightIcon.Layout(gtx, values.MarginPadding20)
+									})
+								})
+							}),
+						)
+					})
+				})
+			}
 			return pg.licenseRow.Layout(gtx, func(gtx C) D {
 				return layout.Flex{}.Layout(gtx,
 					layout.Rigid(func(gtx C) D {
```

Comment:
You are repeating codes. At the very best avoid repeating codes. instead use function and call them where needed.

```suggestion
			licenseRow := func(gtx C) D {
				return pg.licenseRow.Layout(gtx, func(gtx C) D {
					return layout.Flex{}.Layout(gtx,
						layout.Rigid(func(gtx C) D {
							return in.Layout(gtx, pg.license.Layout)
						}),
						layout.Flexed(1, func(gtx C) D {
							return layout.E.Layout(gtx, func(gtx C) D {
								return in.Layout(gtx, func(gtx C) D {
									pg.chevronRightIcon.Color = pg.Theme.Color.Gray1
									return pg.chevronRightIcon.Layout(gtx, values.MarginPadding20)
								})
							})
						}),
					)
				})
			}

			if pg.licenseRow.IsHovered() {
				return pg.shadowBox.Layout(gtx,licenseRow)
			}
			return  licenseRow(gtx)
		},
	}

	return pg.container.Layout(gtx, len(w), func(gtx C, i int) D {
		return layout.Inset{Bottom: values.MarginPadding3}.Layout(gtx, func(gtx C) D {
```
Take note of the insert Bottom  added.